docs/en/README.en.md

```diff
@@ -135,8 +140,32 @@ Then use this command to start Webui:
 ```bash
 python infer-web.py
 ```
+
 If you are using Windows or macOS, you can download and extract `RVC-beta.7z` to use RVC directly by using `go-web.bat` on windows or `sh ./run.sh` on macOS to start Webui.
 
+## ROCm Support for AMD graphic cards (Linux only)
+To use ROCm on Linux install all required drivers as described [here](https://rocm.docs.amd.com/en/latest/deploy/linux/os-native/install.html).
```

Comment:
Worth mentioning that the step "2. Add the AMDGPU Repository and Install the Kernel-mode Driver" can be skipped on most Linux distributions since they'll include the AMD GPU driver. Not only that, but doing this can actually harm your current installation.